firmware/src/src/AR/oled.cpp

```diff
@@ -42,7 +42,7 @@
   }
 #endif
 
-#define POINT_SIZE 6
+#define POINT_SIZE 8
```

Comment:
Won't the shapes become too big?

Reply:
For me it's looks better with a bit bigger size, also please note it's easy to change size base on needs, since it's define (it might change in the future after adding font support)

Reply:
Nothing to add